src/resources/dispatch/DispatchList.tsx

```diff
@@ -50,6 +53,17 @@ const BulkActions = (): React.ReactElement => {
         receiptReceived: nowDate()
       }
     })
+
+    await audit({
```

Comment:
To let users navigate to the dispatch, we need multiple entries, one for each `selectedId` please.  The `id` will go in the `dataIt` field.